src/jquery/data.js

```diff
@@ -1,7 +1,8 @@
 import { migrateWarn } from "../main.js";
 import { camelCase } from "../utils.js";
 
-var oldData = jQuery.data;
+var oldData = jQuery.data,
+ oldFnData = jQuery.fn.data;
```

Comment:
Please use tabs for indentation, e.g. here:
```suggestion
	oldFnData = jQuery.fn.data;
```